web/src/Components/Public/Submission/SubmissionTest/SubmissionTest.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {useStyles} from './SubmissionTest.styles';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import clsx from 'clsx';
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord';
+
+
+export default function SubmissionTest({tests}) {
```

Comment:
Submission test should take the values needed to render one instance of your component not all the tests.